thumb_ip:
  peak: 1.3494800502002051
  valley: 0.0
thumb_mcp:
  peak: 0.7502374462105409
  valley: 0.0
thumb_cmc_flex:
  peak: 0.8479969623113867
  valley: 0.328851980586907
thumb_cmc_abd:
  peak: 1.2489975573087275
  valley: 0.032259727159688
